main.go

```diff
@@ -53,6 +55,11 @@ func main() {
 			Value: "docker",
 		},
 
+		cli.StringFlag{
+			Name:   FlagLocalIP,
```

Comment:
How this suppose to work? I can only know the IP after start the longhorn-manager on k8s cluster.